src/main/webapp/WEB-INF/jsp/login.jsp

```diff
@@ -107,15 +123,17 @@
           </label>
           <span class="col-sm-8"> 
           <input id="keyboard" class="form-control" type='text' name='username' value=''>
+
           </span> 
           <br> <br> 
           <label class="col-sm-4">
             <spring:message	code="label.form.loginPass"> </spring:message>
           </label>
           <span class="col-sm-8"> <input class="form-control"
-            id="keyboard" type='password' name='password' />
+ss            id="keyboard" type='password' name='password' />

```

Comment:
This is added by Saranya for virtual keyboard. Why do u want to remove it??
On Oct 22, 2015 8:32 PM, "Nagarjuna Myla" <notifications@github.com> wrote:

> In src/main/webapp/WEB-INF/jsp/login.jsp
> <https://github.com/chasethenag420/sbs/pull/91#discussion_r42830586>:
>
> >            </span>
> >            <br> <br>
> >            <label class="col-sm-4">
> >              <spring:message	code="label.form.loginPass"> </spring:message>
> >            </label>
> >            <span class="col-sm-8"> <input class="form-control"
> > -            id="keyboard" type='password' name='password' />
> > +ss            id="keyboard" type='password' name='password' />
>
> Remove this.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/chasethenag420/sbs/pull/91/files#r42830586>.
>